OpenRA.Game/ExternalMods.cs

```diff
@@ -80,14 +81,19 @@ void LoadMod(MiniYaml yaml)
 					mod.Icon = sheetBuilder.Add(bitmap);
 			}
 
-			mods.Add(ExternalMod.MakeKey(mod), mod);
+			if (File.Exists(mod.LaunchPath))
+				mods[ExternalMod.MakeKey(mod)] = mod;
```

Comment:
Side effect of this is that deleted mods won't be removed as it loops mods. It could loop .yaml files instead.